### PR TITLE
apiserver crd validation: remove tls1.3 and modern profile

### PR DESCRIPTION
**Pull request by stlaz**
I made some wrong assumptions about turning TLS1.3 on for 1.12 golang binaries...

### CODE REVIEW

**Comment by stlaz** [comment]
@openshift/sig-network-edge please take a look

**Comment by stlaz** [comment]
cc @ironcladlou @Miciah

**Comment by sttts** [comment]
/hold 

until https://github.com/openshift/api/pull/508/files is merged.

File: vendor/k8s.io/kubernetes/openshift-kube-apiserver/admission/customresourcevalidation/apiserver/validate_apiserver.go [hunk 1]
```diff
@@ -219,10 +216,7 @@ func validateCipherSuites(fieldPath *field.Path, suites []string) field.ErrorLis
 
 	if len(suites) > 0 { // something was configured
```
Comment (Miciah) [inline vendor/k8s.io/kubernetes/openshift-kube-apiserver/admission/customresourcevalidation/apiserver/validate_apiserver.go]: Isn't it invalid for `suites` to be empty?

Comment (stlaz) [reply vendor/k8s.io/kubernetes/openshift-kube-apiserver/admission/customresourcevalidation/apiserver/validate_apiserver.go]: Hm, the field does not have `+required` so I was just falling back to intermediate profile ciphers in this case, but the behavior of the field being empty is really not documented. 

I guess forcing it makes sense as any automatic behavior may be counterintuitive.

**Pull request merged**